tests/foreman/api/test_host.py

```diff
@@ -1349,3 +1349,39 @@ def test_positive_verify_files_with_pxegrub2_uefi_secureboot(self):
 
         @caselevel: System
         """
+
+    @tier1
+    def test_positive_read_puppet_proxy_name(self):
+        """Read a hostgroup created with puppet proxy and inspect server's
+        response
+
+        @id: 8825462e-f1dc-4054-b7fb-69c2b10722a2
+
+        @Assert: Field 'puppet_proxy_name' is returned
+
+        @BZ: 1371900
+        """
+        proxy = entities.SmartProxy().search(query={
+            'search': 'url = https://{0}:9090'.format(settings.server.hostname)
+        })[0]
+        host = entities.Host(puppet_proxy=proxy).create().read_json()
```

Comment:
`create()` doing `read()` call automatically, and read() consists from `read_json()`, `read_raw` and etc. Is there any specific reason to have provided call explicitly?
Usually, we try to avoid that